src/components/global/Button/index.js

```diff
@@ -14,6 +14,7 @@ const Button = ({ className, styleType = buttonStyles.primary, ...props }) => {
         {
           [classes.primary]: styleType === buttonStyles.primary,
           [classes.link]: styleType === buttonStyles.link,
+          [classes.menu]: styleType === buttonStyles.menu,
```

Comment:
this could be secondary instead of menu in case we need to reuse this button elsewhere